### PR TITLE
Ability to see info about a table column

### LINKED ISSUE #1992: Ability to see info about a Table column
Part of https://github.com/bigbinary/neeto-engineering-ui-ux/issues/211

@deepakjosp _a

### LINKED ISSUE #2041: Update Table column border style
Ref - https://github.com/bigbinary/neeto-engineering-ui-ux/issues/211#issuecomment-1847676864

@praveen-murali-ind _a

### PR DESCRIPTION
**Pull request by ajmaln**
Fixes #1992 
Fixes #2041

**Description**
Added: Ability to see info about a table column

**Checklist**

- [x] I have made corresponding changes to the documentation.
- [x] I have updated the types definition of modified exports.
- [x] I have verified the functionality in some of the neeto web-apps.
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added the necessary label (patch/minor/major - If package publish
      is required).

**Reviewers**

### CODE REVIEW

**Comment by praveen-murali-ind** [comment]
@anil-anand _a Could you please resolve the conflicts.

**Comment by praveen-murali-ind** [comment]
@ajmaln _a Could you please review.

**Review by ajmaln — COMMENTED** [review]
@akkshaylawrence _a Please check, rest LGTM

File: tests/Table.test.jsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
+/* eslint-disable no-undef */
```
[the remark this answers is not in the document]
Comment (akkshaylawrence) [reply tests/Table.test.jsx]: Removed

**Pull request merged**